CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -57,7 +57,7 @@
     CREATE TABLE IF NOT EXISTS Cases(
         ID INTEGER PRIMARY KEY NOT NULL,
         ReportID INTEGER NOT NULL,
-        CaseID INTEGER, 
+        CaseID TEXT, 
```

Comment:
Great call on changing CaseID to be of a TEXT type in lieu of an INTEGER type - I believe this change was necessary.